pkg/daemon/update.go

```diff
@@ -1076,15 +1078,17 @@ func validateExtensions(exts []string) error {
 }
 
 func (dn *Daemon) applyExtensions(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+	// Right now, we support extensions only on CoreOS nodes
+	if !dn.os.IsCoreOSVariant() {
+		glog.Info("applying extensions on non-CoreOS nodes is not supported on ")
```

Comment:
Did you mean to log the OS variant? `"not supported on " cuts off at the end there

Reply:
ah no. I was rewording sentence to make it consistent with others and it was left as a trail. Thanks for catching this. Fixed.